tasks/main.yml

```diff
@@ -34,3 +34,19 @@
     enabled: true
   tags:
     - prometheus_run
+  when:
+    - not prometheus_docker_install
+
+- name: ensure container is up
+  become: true
+  shell: "{{ prometheus_docker_compose_exec.stdout }} --project-directory {{ prometheus_config_dir }}/docker.d -f {{ prometheus_config_dir }}/docker.d/docker-compose.yml up -d"
```

Comment:
Ensuring that container is up should be done with `docker_container` module

Reply:
ok, I wasn't aware of the docker modules in ansible. This make sense then that docker-compose isn't relevent when working with ansible !
:+1: